Forced -O0 on rtapi/rtapi_pci.cc

Compiling the rtapi part results in a warning:
```
In file included from /usr/include/spawn.h:22,
                 from rtapi/rtapi.h:994,
                 from rtapi/rtapi_pci.cc:34:
/usr/include/features.h:414:4: warning: #warning _FORTIFY_SOURCE requires compiling with optimization (-O) [-Wcpp]
```

The problem is caused by an explicit <code>-O0</code> optimization flag in src/rtapi/Submakefile (line 17). The problem tracks down to the original commit of the PCI interface: https://github.com/LinuxCNC/linuxcnc/commit/96b73e4ed9dc62d864c7ecd2d32636140a614878

My guess is that it has been in there to while developing/debugging the code.

The -O0 flag should be removed and then someone needs to test this on a system with a PCI card. The original commit linked above mentions that the code is for hm2_pci cards to work. Maybe @pcw-mesa or others with a HM2 PCI card can help with testing removal of the -O0 flag and thereby getting rid of the warning?

--- a/src/rtapi/Submakefile
+++ b/src/rtapi/Submakefile
@@ -14,7 +14,7 @@ RTAPI_APP_SRCS := \
 	rtapi/rtapi_pci.cc
 USERSRCS += $(RTAPI_APP_SRCS)
 
-$(call TOOBJSDEPS, rtapi/rtapi_pci.cc): EXTRAFLAGS += $(LIBUDEV_CFLAGS) -O0
+$(call TOOBJSDEPS, rtapi/rtapi_pci.cc): EXTRAFLAGS += $(LIBUDEV_CFLAGS)
 $(call TOOBJSDEPS, $(RTAPI_APP_SRCS)): EXTRAFLAGS += -DSIM \
 	-UULAPI -DRTAPI -pthread
 ../bin/rtapi_app: $(call TOOBJS, $(RTAPI_APP_SRCS))

